lib/private/Setup.php

```diff
@@ -542,19 +542,27 @@ public static function protectDataDirectory() {
 		//Require all denied
 		$now =  date('Y-m-d H:i:s');
 		$content = "# Generated by Nextcloud on $now\n";
-		$content.= "# line below if for Apache 2.4\n";
+		$content.= "# Section for Apache 2.4 and 2.5\n";
 		$content.= "<ifModule mod_authz_core.c>\n";
-		$content.= "Require all denied\n";
+		$content.= "  Require all denied\n";
+		$content.= "</ifModule>\n";
+		$content.= "<ifModule mod_access_compat.c>\n";
+		$content.= "  Deny from all\n";
+		$content.= "  Satisfy All\n";
 		$content.= "</ifModule>\n\n";
-		$content.= "# line below if for Apache 2.2\n";
+		$content.= "# Section for Apache 2.2\n";
 		$content.= "<ifModule !mod_authz_core.c>\n";
```

Comment:
```suggestion
		$content.= "<IfModule !mod_authz_core.c>\n";
```

https://httpd.apache.org/docs/2.4/mod/core.html#ifmodule